src/components/EditListingWizard/EditListingWizard.js

```diff
@@ -263,6 +272,8 @@ class EditListingWizard extends Component {
       fetchStripeAccountError,
       stripeAccountFetched,
       stripeAccount,
+      stripeAccountError,
+      stripeAccountLinkError,
```

Comment:
When adding new props to component we should also make sure that these new props are added to propType check in the end of this file. I have really bad habit of forgotting these myself so propably I didn't mention it yesterday.

PropTypes are checked here: https://github.com/sharetribe/ftw-daily/blob/b5ad36f5427f1d664e021ce86b3118bb69a3c9a0/src/components/EditListingWizard/EditListingWizard.js#L476
These new props should not be required so we can also add default value (null) for them: 
https://github.com/sharetribe/ftw-daily/blob/b5ad36f5427f1d664e021ce86b3118bb69a3c9a0/src/components/EditListingWizard/EditListingWizard.js#L469

We should check the propTypes also on `EditListingPage` and `PayoutDetailsPage`